gurubu-client/src/app/page.tsx

```diff
@@ -10,6 +11,7 @@ export default function Home() {
       <Navbar />
       <Greeting />
       <Howto />
+      <Testimonial />
```

Comment:
@armagandalkiran Would it be healthy to temporarily merge this into the comment line?
```suggestion
      //<Testimonial />
```